src/components/performance/cell-status-mixin.cjsx

```diff
@@ -0,0 +1,35 @@
+React = require 'react'
+Router = require 'react-router'
+LateIcon = require './late-icon'
+
+module.exports = {
+
+  propTypes:
+    courseId: React.PropTypes.string.isRequired
+
+    task: React.PropTypes.shape(
+      status:          React.PropTypes.string
+      due_at:          React.PropTypes.string
+      last_worked_at:  React.PropTypes.string
+      type:            React.PropTypes.string
+    ).isRequired
+
+    student: React.PropTypes.shape(
+      name: React.PropTypes.string
+      role: React.PropTypes.number
+    ).isRequired
+
+  isLate: ->
+    # if the task was never worked then it's not late, it's incomplete
+    return false unless @props.task.last_worked_at
+
+    new Date(@props.task.last_worked_at) > new Date(@props.task.due_at)
+
```

Comment:
Some late checking logic here as here (I want to take the homework out of it).  https://github.com/openstax/tutor-js/pull/498/files#diff-74c51a44b8ba99c878804b91182ab691R202

Would be nice to consolidate and either keep using this ```isLate``` or the TaskStore's ```getStepLateness``` but we can probably get to that in a different PR